pkg/subctl/cmd/gather/servicediscovery.go

```diff
@@ -110,10 +116,44 @@ func gatherConfigMapCoreDNS(info *Info) {
 	}
 }
 
+// gatherInternalServices gathers the internal service created for every exported submariner service.
+func gatherInternalServices(info *Info) {
+	services, err := info.ClientProducer.ForKubernetes().CoreV1().Services(corev1.NamespaceAll).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		info.Status.Warning("Error gathering the internal service configuration: %v", err)
+	}
+
+	for k := range services.Items {
+		namespace := services.Items[k].Namespace
+		if !(strings.Contains(namespace, names.OperatorImage) || strings.Contains(namespace, "kube-system")) {
+			label := map[string]string{internalSvcLabel: services.Items[k].Name}
+			labelSelector := labels.Set(label).String()
+			service, err := info.ClientProducer.ForKubernetes().CoreV1().Services(namespace).List(context.TODO(),
+				metav1.ListOptions{LabelSelector: labelSelector})
+
```

Comment:
`golangci-lint run` wants a space and `gofumpt -w servicediscovery.go` doesn't want a space...